mmv1/third_party/terraform/services/securitycentermanagement/resource_scc_management_organization_event_threat_detection_custom_module_test.go.erb

```diff
@@ -45,7 +45,7 @@ func testAccSecurityCenterManagementOrganizationEventThreatDetectionCustomModule
 
 	acctest.VcrTest(t, resource.TestCase{
 		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
-		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderBetaFactories(t), // Ensure beta factory is used
```

Comment:
`acctest.ProtoV5ProviderBetaFactories` need to be used to the tests in the other three files as well.

Reply:
Addressed.